spec/unit/provider/directory_spec.rb

```diff
@@ -197,6 +197,37 @@
         expect { directory.run_action(:create) }.to raise_error(Chef::Exceptions::EnclosingDirectoryDoesNotExist)
       end
     end
+
+    describe "on OS X", mac_osx_only: true do
```

Comment:
Lack of knowledge on my part - because we're mocking out the `Dir` responses below, do we need `mac_osx_only: true` here?  My thought is that since we're mocking we could run this test on all platforms.  Can someone with more core chef spec knowledge chime in here?  Is `mac_osx_only` for integration tests?